vector/src/main/java/im/vector/adapters/VectorParticipantsAdapter.java

```diff
@@ -1030,70 +1030,66 @@ public View getChildView(int groupPosition, int childPosition, boolean isLastChi
-            //thumbView.setAlpha( 0.5f);
-            nameTextView.setTypeface(null, Typeface.ITALIC);
-        } else {
-            //thumbView.clearColorFilter();
+        // Tchap users are displayed differently than no Tchap users
+        if (participant.isMatrixUser()){
+            onlineStatusImageView.setVisibility(VectorUtils.isUserOnline(mContext, mSession, participant.mUserId, null) ? View.VISIBLE : View.GONE);
+            domainNameTextView.setVisibility(View.VISIBLE);
+            domainNameTextView.setPadding(0, 54, 0, 0);
+            nameTextView.setText(DinsicUtils.getNameFromDisplayName(participant.mDisplayName));
+            nameTextView.setPadding(0, 37, 0, 0);
             nameTextView.setTypeface(null, Typeface.BOLD);
+            emailTextView.setVisibility(View.GONE);
+        } else {
+            domainNameTextView.setVisibility(View.GONE);
+            domainNameTextView.setPadding(0, 0, 0, 0);
+            nameTextView.setText(participant.mDisplayName);
+            nameTextView.setPadding(0, 0, 0, 0);
+            nameTextView.setTypeface(null, Typeface.ITALIC);
+            emailTextView.setVisibility(View.VISIBLE);
+            // We display an additional information like email or phone number.
+            if (participant.mContact.getEmails().size() > 0) {
+                emailTextView.setText(participant.mContact.getEmails().get(0));
+            } else if (participant.mContact.getPhonenumbers().size() > 0) {
+                emailTextView.setText(participant.mContact.getPhonenumbers().get(0).mRawPhoneNumber);
+            }
         }
 
-        // display the avatar
+        // display the participant's avatar
         participant.displayAvatar(mSession, thumbView);
 
-        synchronized (LOG_TAG) {
-            nameTextView.setText(DinsicUtils.getNameFromDisplayName(participant.getUniqueDisplayName(mDisplayNamesList)));
-            domainNameTextView.setText(DinsicUtils.getDomainFromDisplayName(participant.getUniqueDisplayName(mDisplayNamesList)));
-        }
-
-        // set the presence
-        String status = "";
-
-        if (groupPosition == mKnownContactsSectionPosition) {
-            User user = null;
-            MXSession matchedSession = null;
-            // retrieve the linked user
-            ArrayList<MXSession> sessions = Matrix.getMXSessions(mContext);
-
-            for (MXSession session : sessions) {
-                if (null == user) {
-                    matchedSession = session;
-                    user = session.getDataHandler().getUser(participant.mUserId);
-                }
-            }
-
-            if (null != user) {
-                status = VectorUtils.getUserOnlineStatus(mContext, matchedSession, participant.mUserId, new SimpleApiCallback<Void>() {
-                    @Override
-                    public void onSuccess(Void info) {
-                        VectorParticipantsAdapter.this.refresh(mFirstEntry, null);
+        // display the participant's domain
```

Comment:
Move the participant's domain handling under the condition "if (participant.isMatrixUser())"